packages/claude-plugin/README.md

```
@@ -1,142 +1,125 @@
-pnpm --filter @storybook/claude-code-plugin run plugin:list
-```
+1. Run this command to remove the Storybook marketplace, which will also uninstall the plugin:
 
-The output should include `storybook@storybook` with:
+   ```bash
+   claude plugin marketplace remove storybook --scope user
+   ```
 
-- `"scope": "user"`
-- `"enabled": true`
+2. Follow the [installation instructions above](#claude-code) to re-add the marketplace and install the plugin again.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)